client/components/FirstVendorStoreFront.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
```

Comment:
Need to sync our linters, cause there are a lot of linter issues. -unused imports/variables, self-enclosing components, semi-colons, spacing

Reply:
okay! Also i was under the impression we are using semi colons.. maybe i misunderstood, lets sync our linters tomorrow I am fixing your other suggestions right now, thanks!

Reply:
Ohhhh okay, I forgot. Yeah let's work it out tomorrow then.